src/render/rt64_shader_compiler.h

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#if defined(_WIN32)
+#if defined(_WIN32) && !defined(RT64_BUILD_PLUGIN)
```

Comment:
It doesn't sound like there should be a reason to _not_ include the shader compiler for Plugin builds. Plugin builds on Windows will and should include D3D12 support.

Reply:
I need mingw support for RMG to include it in that build system though, and currently that fails to build, hence why I disabled it for now.

Reply:
Do the mingw build problems look solvable? I imagine that's not the case. I'd suggest swapping out all the instances of something like this to check for mingw instead of whether it's a plugin build or not.

Reply:
It was a lot of errors so not easily, and a mingw check indeed seems more feasable and better, I'll try to get that changed soon.

Reply:
That matches my experience to the point of the build problems being unsolvable for D3D12, one of the reasons I had to split off RT64 for the sm64pc ports. The headers are just not very well maintained. I suggest just not supporting D3D12 under mingw as the long term solution.